### PR TITLE
created the event_members controller and routes

### PR DESCRIPTION
**Pull request by Aremusch**

### CODE REVIEW

File: app/controllers/event_members_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if @event_member.save
+      redirect_to @guild.find(params[:id])
+    else
+      root_path
+    end
+  end
+
+  def destroy
+    @event_member.destroy
+    redirect_back(fallback_location: root_path)
```
Comment (williamgrant04) [inline app/controllers/event_members_controller.rb]: I think this fallback should be the guild path

File: config/routes.rb [hunk 2]
```diff
@@ -4,7 +4,9 @@
     sessions: "users/sessions"
   }
   resources :guilds, except: %i[destroy index] do
-    resources :games, only: %i[new create]
+    resources :games, only: %i[new create] do
+      resources :event_member, only: %i[new create destroy]
```
Comment (williamgrant04) [inline config/routes.rb]: I don't think we need to nest the event members

**Review by williamgrant04 — CHANGES_REQUESTED** [review]

File: app/controllers/event_members_controller.rb (hunk 1 above)
Comment (Aremusch) [reply app/controllers/event_members_controller.rb]: changes done

File: config/routes.rb (hunk 2 above)
Comment (Aremusch) [reply config/routes.rb]: changes done

File: app/controllers/event_members_controller.rb [hunk 3]
```diff
@@ -15,7 +15,7 @@ def create
 
   def destroy
     @event_member.destroy
-    redirect_back(fallback_location: root_path)
+    redirect_back(fallback_location: @guild_path)
```
Comment (williamgrant04) [inline app/controllers/event_members_controller.rb]: `guild_path(@guild)`

**Review by williamgrant04 — CHANGES_REQUESTED** [review]

File: app/controllers/event_members_controller.rb (hunk 3 above)
Comment (Aremusch) [reply app/controllers/event_members_controller.rb]: change done

**Review by williamgrant04 — APPROVED** [review]

**Pull request merged**